polaris/ocean/omega.cfg

```diff
@@ -0,0 +1,24 @@
+# This config file has default config options for Omega
+
+# The paths section points polaris to external paths
+[paths]
+
+
+# the relative or absolute path to the Omega component source code
+source_path = ${paths:polaris_branch}/e3sm_submodules/Omega/components/omega
+
+# the relative or absolute path to the root of a branch where Omega
+# has been built
+component_path = ${paths:source_path}/build
+
+# The model_config section defines paths to yaml file with default model config
+# options that will be used to generate specific yaml config files for Omega.
+[model_config]
+defaults = ${paths:source_path}/configs/Default.yml
```

Comment:
This didn't work as desired.  The `-p` flag only changes `component_path` so `source_path` stays pointing to the default location.  This will need a fix requested in https://github.com/E3SM-Project/Omega/issues/126 and then to be updated.

Reply:
```suggestion
# The model_config section defines paths to yaml file with default model config
# options that will be used to generate specific yaml config files for Omega.
[model_config]
defaults = ${paths:component_path}/configs/Default.yml
```